cas-server-integration-ehcache/src/main/java/org/jasig/cas/ticket/registry/EhCacheTicketRegistry.java

```diff
@@ -50,7 +51,7 @@
  * @author Andrew Tillinghast
  * @since 3.5
  */
-public final class EhCacheTicketRegistry extends AbstractDistributedTicketRegistry implements InitializingBean {
+public final class EhCacheTicketRegistry extends AbstractCrypticTicketRegistry implements InitializingBean {

```

Comment:
That's exactly correct. It only applies to distributes ticket registries that can do serialization, such ehcache, hazelcast, jboss and memached. I suspect it may also work with nosql-based registries such as couchdb and mongo, if we ever develop one. I have included sample snippets in the ticket registry xml file to show how a cipher can be added and this of course will be documented.

Glad we came to a resolution on this. Thanks for bearing with me.